samples/CMakeLists.txt

```diff
@@ -143,3 +143,35 @@ if (CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE)
     DOMAIN CPUAPP
     BOARD ${CONFIG_DOMAIN_CPUAPP_BOARD})
 endif()
+
+if (CONFIG_BT_RPC)
+  if (CONFIG_SOC_NRF5340_CPUAPP)
+    message("Adding 'rpc_host' firmware as child image since "
+      "CONFIG_BT_RPC is set to 'y'")
+
+      if (CONFIG_BOOTLOADER_MCUBOOT AND CONFIG_NRF53_UPGRADE_NETWORK_CORE)
```

Comment:
Could we extract this into a macro? This file has already gotten out of control, so we should strive to avoid duplication whenever we can. AFAICS its an exact copy of the handling for `CONFIG_BT_RPMSG_NRF53` case.

Reply:
Yes I want this sample to be added in the same way like the BT_RPMSG_NRF53. I think I can try to move this condition into a macro. You can also give me some suggestions if my approach is valid here ;)

Reply:
Ok. Done